### PR TITLE
Fix exceptions on no service file found

### PR DESCRIPTION
**Pull request by signedav**
Catch the Exceptions from pgserviceparser when no service file availble or service not found.

### CODE REVIEW

**Comment by gacarrillor** [comment]
@signedav, is there any related issue?
Yesterday I got one (I'll create the issue right now), and I wonder if this PR would solve it.

**Comment by signedav** [comment]
Reported by mail

**Comment by gacarrillor** [comment]
> Reported by mail

Perhaps it's the same issue? https://github.com/opengisch/QgisModelBaker/issues/944

File: QgisModelBaker/gui/panel/pg_config_panel.py [hunk 1]
```diff
@@ -223,7 +227,10 @@ def get_fields(self, configuration):
     @override
     def set_fields(self, configuration):
 
-        if configuration.dbservice is None:
+        service_config = self._get_service_config(configuration.dbservice)
+
+        # if no dbservice in the configuration or one is there but the the servicefile is not available anymore
```
Comment (gacarrillor) [inline QgisModelBaker/gui/panel/pg_config_panel.py:232]: typo: `the the`

**Review by gacarrillor — APPROVED** [review]
Looks good to me.

**Comment by signedav** [comment]
> > Reported by mail
> 
> Perhaps it's the same issue? #944

Same issue, yes, but not solved here. I need to make another one in the library. Will trigger you.

**Comment by signedav** [comment]
@gacarrillor see https://github.com/opengisch/QgisModelBakerLibrary/pull/104

**Review by gacarrillor — APPROVED** [review]
I just double checked after moving the logic to the lib.

**Pull request merged**